tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -26,9 +26,9 @@ HRESULT CInstrumentationMethod::Initialize(_In_ IProfilerManager* pProfilerManag
         pProfilerManager->GetCorProfilerInfo((IUnknown**)&pCorProfilerInfo);
         pCorProfilerInfo->SetEventMask(COR_PRF_MONITOR_EXCEPTIONS | COR_PRF_ENABLE_STACK_SNAPSHOT);
 
-        CComPtr<IProfilerManagerLogging> spLogger;
-        pProfilerManager->GetLoggingInstance(&spLogger);
-        spLogger->LogDumpMessage(_T("<ExceptionTrace>\r\n"));
+        CComQIPtr<IProfilerManager4> pProfilerManager4 = pProfilerManager;
```

Comment:
Why was this changed?

Reply:
The GetLoggingInstance appends the IM prefix, however the GetGlobalLoggingInstance does not. This was the quickest way to unblock the Tests since they assume the logging output generates correct XML (the IM prefix breaks this).

I'll add IM-specific logging tests in the next iteration

---
In reply to: [333780324](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r333780324) [](ancestors = 333780324)

Reply:
Turns out there was a bug in the XML comparison (it compared the xml to itself so always succeeds). 

---
In reply to: [336672890](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r336672890) [](ancestors = 336672890,333780324)

Reply:
Could you revert this?

Reply:
Sure

---
In reply to: [341218858](https://github.com/microsoft/CLRInstrumentationEngine/pull/146#discussion_r341218858) [](ancestors = 341218858)